item.rb

```diff
@@ -1,11 +1,13 @@
 require 'date'
+require './label'
```

Comment:
Consider maintaining consistency regarding import statements.
Change `require './label'` to `require_relative 'label' `